lib/mi_bridges/driver/base_page.rb

```diff
@@ -5,7 +5,7 @@ class Driver
     class BasePage
       include Capybara::DSL
 
-      def initialize(snap_application, logger:)
+      def initialize(snap_application, logger: nil)
```

Comment:
Is there a default logger which is more appropriate to set here than `nil`?

Aside: `nil` and I are not friends.

Reply:
We could create a NullLogger. 

The reason I did this was that I was constantly instantiating pages in rails console and getting a valid logger each time was a lot of typing and remembering each time. We could have a smarter default, but I am not 100% sure if there is a large benefit to that, since we only do one nil check on this...

Reply:
I like the `NullLogger`, but I love all null objects.

To me, I feel like `nil` always multiplies so I try to root it out whenever I see it. Not a huge deal though, I'm a 2/10 on this opinion.

Reply:
I would def go for a null object if we were going to pass this around in multiple places, but for now it's just the one so I am OK with a nil check.